src/styles/blocks/stars.css

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--1 .stars__star:nth-child(-n+1),
```

Comment:
and so on
```suggestion
.stars--1 :nth-child(-n+1),
```

Reply:
I cannot create it like your advice cause class nth-child work only with father element, not with modificator